ccm_web/package.json

```diff
@@ -26,9 +26,13 @@
     "@nestjs/config": "^0.6.3",
     "@nestjs/core": "^7.6.15",
     "@nestjs/platform-express": "^7.6.15",
-    "@nestjs/swagger": "^4.8.0",
     "@nestjs/sequelize": "^0.2.0",
+    "@nestjs/swagger": "^4.8.0",
+    "class-transformer": "^0.4.0",
+    "class-validator": "^0.13.1",
+    "connect-session-sequelize": "^7.1.1",
     "express": "^4.17.1",
+    "express-session": "^1.17.2",
```

Comment:
Is this module the key to keeping the user auth. info in a session for future requests?

Reply:
`express-session` and the store library `connect-session-sequelize` provide the functionality needed to associate custom data with a user's session.